requirements/sso-requirements.in

```diff
@@ -1,3 +1,4 @@
 pyOpenSSL>=23.2.0  # for cryptography>=41
 cryptography>=41   # for security vulnerability
 python3-saml
+msal
```

Comment:
```suggestion
msal

```

Reply:
new line at end of file nit

Reply:
Thank you Biyeun! Fixed here: a86ca6cb8243f2bf98b40ab643db861df72e7099 (I should just commit suggestion)